lumicks/pylake/force_calibration/tests/test_driving_input.py

```diff
@@ -0,0 +1,32 @@
+from lumicks.pylake.force_calibration.detail.driving_input import estimate_driving_input_parameters
+import numpy as np
+import pytest
+
+
+@pytest.mark.parametrize(
+    "amps, frequencies, true_amp, true_freq, guess",
+    [
+        [[0.5, 1], [36.9, 50], 0.5, 36.9, 36],
```

Comment:
I find the data structure here curious.
I would have used a list of amplitude/frequency pairs, so instead of passing the "true" amplitude and frequency again, we could use an index.

``` python
        [[0.5, 36.9], [1, 50], 0, 36],
```

Reply:
Will do!